package.json

```diff
@@ -15,7 +15,7 @@
   "author": "Nathan Rajlich <nathan@tootallnate.net> (http://tootallnate.net)",
   "repository": {
     "type": "git",
-    "url": "git://github.com/TooTallNate/node-gyp.git"
+    "url": "git://github.com/rvagg/pan-gyp.git"
```

Comment:
derp, I temporarily released `pangyp` while we wait for this merge, this line shouldn't have made it here and is about to disappear as soon as I stop typing this